### ISSUE TITLE
Mode question - answer 

### ISSUE THREAD

**Issue by DanteAdamss** [macroxue/2048-ai#5]
Hi, Macroxue. THX for this amazing work. I have question for you.  Can i run this script in mode when i send to console command right with massive like this  [16,512,2,0,64,256,0,2,16,128,0,0,8,4,0,0] and got best direction?  This is very needing function for me.

**Comment by macroxue** [comment]
Yes, you can. Please see [Server mode](https://github.com/macroxue/2048-ai#server-mode).

Once the AI is running in server mode with `./2048b -S 8080 -I -v`, you can issue an HTTP get request and pass in the board layout. For example, `wget -qO- http://localhost:8080/move?board=EDC1BA9187611110; echo` returns `u`, suggesting an up move for the following board.
```
---------------------------------
| 16384 |  8192 |  4096 |     2 |
---------------------------------
|  2048 |  1024 |   512 |     2 |
---------------------------------
|   256 |   128 |    64 |     2 |
---------------------------------
|     2 |     2 |     2 |       |
---------------------------------
```

**Comment by DanteAdamss** [comment]
THX for your answering it's help me strongly. I tried solving some modified version of 2048 where after some needing score value  reaching board starts generate new cell with 8 and 16 value. It leads that AI solve this type of game worse than classic  2048. Maybe, I can or You can do some small modification on game where AI should get  their solving base for do it better to this type of game?

**Comment by macroxue** [comment]
It would require a lot of changes to handle the spawning of 8 and 16 tiles. These tiles definitely make the game harder than the classic 2048. Below are a few places that require changes. You can give it a try. Good luck.

https://github.com/macroxue/2048-ai/blob/master/tuple.h#L242
https://github.com/macroxue/2048-ai/blob/master/node.h#L114
https://github.com/macroxue/2048-ai/blob/master/2048.cc#L181
https://github.com/macroxue/2048-ai/blob/master/plan.h#L69

**Comment by DanteAdamss** [comment]
THX

**Comment by alpha-106** [comment]
Thanks, its working for me now. However I find it quite tedious to input the state every move, can I use any other method to update the state?

**Comment by macroxue** [comment]
You can update your program to get moves from the AI. Please check this [2048 clone](https://github.com/macroxue/2048-clone) which has been modified to work with the AI server. You can look at  this [commit](https://github.com/gabrielecirulli/2048/commit/0f1aa229e8efa98fd7e36d8e3b791f8a68e5f649) for details.

**Comment by bighohoo** [comment]
> THX for your answering it's help me strongly. I tried solving some modified version of 2048 where after some needing score value reaching board starts generate new cell with 8 and 16 value. It leads that AI solve this type of game worse than classic 2048. Maybe, I can or You can do some small modification on game where AI should get their solving base for do it better to this type of game?

What was the result? Was it successful on site? I just encountered the same problem now.
